src/main/java/com/puppycrawl/tools/checkstyle/api/JavadocTokenTypes.java

```diff
@@ -1206,7 +1204,7 @@
     /**
      * End Of File symbol.
      */
-    public static final int EOF = Recognizer.EOF;
+    public static final int EOF = -1;
```

Comment:
There is no other way to remove antlr then to give it a constant.

Reply:
lets put this in comment, to explain magic number

`value org.antlr.v4.runtime.Recognizer.EOF, copied to avoid dependency to antlr`

Reply:
Done.